plugins/destination/snowflake/README.md

```diff
@@ -0,0 +1,38 @@
+# CloudQuery Snowflake Destination Plugin
+
+The snowflake plugin helps you sync data to your Snowflake data warehouse.
+
+There are two ways to sync data to Snowflake:
+
+1. Direct (easy but not recommended for production or large data sets): This is the default mode of operation where CQ plugin will stream the results directly to the snowflake database. There is no additional setup needed apart from authentication to Snowflake.
+
+2. Loading via CSV/Json from a remote Storage: This is a standard way of loading data into Snowflake, it is recommended for production and large data sets. This mode requires a remote storage (e.g. S3, GCS, Azure Blob Storage) and a Snowflake stage to be created. The CQ plugin will stream the results to the remote storage. You can then load those files via a cronjob or via SnowPipe. This method is still in the works and will be updated soon with a guide.
```

Comment:
```suggestion
2. Loading via CSV/JSON from a remote storage: This is the standard way of loading data into Snowflake, it is recommended for production and large data sets. This mode requires a remote storage (e.g. S3, GCS, Azure Blob Storage) and a Snowflake stage to be created. The CQ plugin will stream the results to the remote storage. You can then load those files via a cronjob or via SnowPipe. This method is still in the works and will be updated soon with a guide.
```

BTW with snowflake you don't really need stages, `COPY INTO` can directly refer to an S3 bucket for instance (but you'd need access_key_id in the command which you might not want)

Reply:
I do think you still need stages at least per what I read recently even with remote storage you need to specify a stage first and mention that this stage related to a specific remote storage.